src/feat/signal-test.cc

```diff
@@ -0,0 +1,47 @@
+// feat/signal-test.cc
+
+// Copyright 2015  Tom Ko
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#include "base/kaldi-common.h"
+#include "util/common-utils.h"
+#include "feat/signal.h"
+
+namespace kaldi {
+
```

Comment:
I was just explaining why you should implement the simple convolution.  No
further action required.

On Fri, Sep 4, 2015 at 12:07 AM, tomkocse <notifications@github.com> wrote:

> In src/feat/signal-test.cc
> <https://github.com/kaldi-asr/kaldi/pull/76#discussion_r38720259>:
>
> > +//
> > +//  http://www.apache.org/licenses/LICENSE-2.0
> > +//
> > +// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
> > +// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
> > +// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
> > +// MERCHANTABLITY OR NON-INFRINGEMENT.
> > +// See the Apache 2 License for the specific language governing permissions and
> > +// limitations under the License.
> > +
> > +#include "base/kaldi-common.h"
> > +#include "util/common-utils.h"
> > +#include "feat/signal.h"
> > +
> > +namespace kaldi {
> > +
>
> "Your testing method does not test for bugs that would affect both of the
> FFT-based methods (e.g. problems with the sign, or a scalar factor that
> shouldn't be there)." <---
> Can you elaborate this and explicitly tell what cases should be tested?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/76/files#r38720259>.
>